controllers/bsl.go

```diff
@@ -283,8 +283,9 @@ func (r *DPAReconciler) validateAWSBackupStorageLocation(bslSpec velerov1.Backup
 	if len(bslSpec.StorageType.ObjectStorage.Prefix) == 0 && dpa.BackupImages() {
 		return fmt.Errorf("prefix for AWS backupstoragelocation object storage cannot be empty. It is required for backing up images")
 	}
+
 	// BSL region is required when s3ForcePathStyle is true AND BackupImages is true
```

Comment:
change comment AND to or to match if behavior? Add TODO (or create a GitHub issue, no JIRA, just for tracking) about need to change when https://github.com/openshift/openshift-velero-plugin/pull/223 merges?

Reply:
Done